goodtablesio/plugins/github/blueprint.py

```diff
@@ -2,18 +2,99 @@
 import logging
 
 from celery import chain
-from flask import Blueprint, request, abort, jsonify
+from flask import Blueprint, request, abort, session
+from flask import render_template, jsonify, redirect, url_for
+from flask_login import login_required
+
+from goodtablesio import models, settings
+from goodtablesio.services import database
+from goodtablesio.tasks.validate import validate
+from goodtablesio.plugins.github.models.repo import GithubRepo
+from goodtablesio.plugins.github.tasks.jobconf import get_validation_conf
+from goodtablesio.plugins.github.tasks.repos import sync_user_repos
+from goodtablesio.plugins.github.utils.status import set_commit_status
+from goodtablesio.plugins.github.utils.signature import validate_signature
+from goodtablesio.plugins.github.utils.hook import (
+    activate_hook, deactivate_hook, get_owner_repo_sha_from_hook_payload)
 
-from goodtablesio import tasks, models, settings
 
-from goodtablesio.plugins.github.tasks import get_validation_conf
-from goodtablesio.plugins.github.utils import set_commit_status, validate_signature
+log = logging.getLogger(__name__)
 
 
-log = logging.getLogger(__name__)
+github = Blueprint('github', __name__, url_prefix='/github', template_folder='templates')
+
+
+@github.route('/')
+def home():
+
+    # Get github syncing status
+    sync = False
+    if session.get('github_sync_task_id'):
```

Comment:
Yes, I think we need to move it to the database (first see how it will be going with other plugins). But it looks like we need it in any case:
- user registers
- user goes to github repos
- user click sync repos
- then we need to know status of syncing job to show `in-progress` or updated list of repos. Without it we don't have enough information about repos update (still pending/finished)